automation/repeated_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+SCRIPT_PATH="$(
+    cd "$(dirname "$BASH_SOURCE[0]")/"
+    echo "$(pwd)/"
+)"
+
+NUM_TESTS=${NUM_TESTS:-3}
+if [[ -z ${NEW_TESTS} ]]; then
```

Comment:
Personally, I usually prefer a script with a proper CLI instead of implicitly using env vars... I just think that it's less confusing... but, I know that this is just a personal preference so feel free to not address this comment.